components/partials/header.js

```diff
@@ -13,6 +13,7 @@ export default function Header({ mixpanel }) {
   const [showHeader, setShowHeader] = useState(true);
   var [lastScrollPos, setLastScrollPos] = useState(0);
   const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const [showContributionMenu, setShowContributionMenu] = useState(false);
```

Comment:
Issue:
- Hover on contribution and it will show the submenu
- Hover on career, but the submenu is still open.

It should be hidden